api/src/test/java/io/kaoto/backend/api/resource/v1/IntegrationsResourceTest.java

```diff
@@ -172,25 +169,93 @@ void complexEIP() throws URISyntaxException, IOException {
     @Test
     void amqAmq() throws Exception {
         String yaml = Files.readString(Path.of(
```

Comment:
I also made this test stronger while trying to understand where the bug comes from. So win-win.